src/Comment/components/Comment.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import '../styles/Comment.css';
+
+const Comment = (props) => (
+    <div className="comment-body">
+        { props.commentBody }
```

Comment:
{} can be omitted

Reply:
The {} are so that react renders it as the variable commentBody in props. Without the {}, it thinks it is the string "props.commentBody". I'm not sure if there is a way to avoid this without the {}